### PR TITLE
Support Custom VPC Tag Lookup

### PR DESCRIPTION
**Pull request by dennybaa**
## what
* Support looking up kops clusters by arbitrary tags and values

## why
* Better way to do it

### CODE REVIEW

**Comment by dennybaa** [comment]
well np, rebased.

File: variables.tf [hunk 1]
```diff
@@ -25,3 +25,13 @@ variable "nodes_name" {
   default     = "nodes"
   description = "K8s nodes subdomain name in the Kops DNS zone"
 }
+
+variable "vpc_tag" {
+  default     = "Name"
+  description = "Use Name tag to lookup the Kops VPC"
```
Comment (osterman) [inline variables.tf]: This description is now misleading. It should probably say "Tag used to lookup the Kops VPC"

File: variables.tf [hunk 2]
```diff
@@ -25,3 +25,13 @@ variable "nodes_name" {
...
+}
+
+variable "vpc_tag_values" {
+  default     = []
+  description = "Provide tag value to lookup the Kops VPC"
```
Comment (osterman) [inline variables.tf]: "Tag value used to lookup the Kops VPC"

**Comment by dennybaa** [comment]
Now ok?

**Comment by dennybaa** [comment]
@osterman welcome and likewise thank you for useful modules :+1:

**Comment by osterman** [comment]
@dennybaa my pleasure! stop by our slack team sometime. https://slack.cloudposse.com

**Pull request merged**